dojo/api_v2/serializers.py

```diff
@@ -2130,6 +2130,11 @@ class ImportScanSerializer(serializers.Serializer):
     product_type_id = serializers.IntegerField(read_only=True)
 
     statistics = ImportStatisticsSerializer(read_only=True, required=False)
+    apply_tags_to_findings = serializers.BooleanField(
+        help_text="If set to True, the tags will be applied to the findings",
+        required=False,
+        default=True,
+    )
```

Comment:
How is this new variable being used in the `import_scan` and `reimport_scan` functions?